anubis-management-api/src/policies/operations.py

```diff
@@ -108,13 +121,23 @@ def get_policies_by_agent(
 def _get_policies_by_service_path(
         db: Session,
         service_path_id: str,
+        agent_type: str = None,
         resource: str = None,
         resource_type: str = None,
         skip: int = 0,
         limit: int = 100):
     db_policies = db.query(
         models.Policy).filter(
         models.Policy.service_path_id == service_path_id)
+    if agent_type:
+        if agent_type in default.DEFAULT_AGENTS:
+            db_policies = db_policies.join(
+                models.Policy.agent).filter(
+                    models.Agent.iri.startswith(agent_type))
+        if agent_type in default.DEFAULT_AGENT_TYPES:
+            db_policies = db_policies.join(
+                models.Policy.agent).filter(
+                    models.Agent.iri.startswith(agent_type + ":"))
     if resource:
         db_policies = db_policies.filter(models.Policy.access_to == resource)
```

Comment:
maybe here a "like" rather than == would be more useful, feelings?

Reply:
See above